app/views/email/CaptureEmailView.scala.html

```diff
@@ -14,69 +14,90 @@
  * limitations under the License.
  *@
 
-@import uk.gov.hmrc.play.views.html.helpers.FormWithCSRF
 @import config.AppConfig
 @import views.html.templates.inputs.Text
 @import views.html.templates.errors.ErrorSummary
 @import play.api.mvc.Call
 
-@this(mainTemplate: MainTemplateOriginal, form: FormWithCSRF, errorSummary: ErrorSummary, text: Text)
+@this(mainTemplate: MainTemplate, form: FormWithCSRF, govukErrorSummary: GovukErrorSummary, govukButton: GovukButton,
+govukBackLink: GovukBackLink, text: Text, govukInput : GovukInput, govukDetails : GovukDetails, formWithCSRF: FormWithCSRF)
```

Comment:
Can you remove the `Text` inject from here as this is the old text input component that will be deleted soon